homeassistant/components/enocean/sensor.py

```diff
@@ -82,6 +109,15 @@
         vol.Optional(CONF_MIN_TEMP, default=0): vol.Coerce(int),
         vol.Optional(CONF_RANGE_FROM, default=255): cv.positive_int,
         vol.Optional(CONF_RANGE_TO, default=0): cv.positive_int,
+        vol.Optional(CONF_DATA_BYTE, default=3): cv.positive_int,
```

Comment:
I don't think we want to extend the config schema until it's decided how the future of Enocean in Home Assistant should be. We introduced a config flow but didn't remove the support for YAML configuration. See the preview us Enocean config flow PR for that discussion.